parasolr/query/queryset.py

```diff
@@ -213,17 +209,26 @@ def get_facets(self) -> Dict[str, Dict]:
     def get_stats(self) -> Optional[Dict[str, ParasolrDict]]:
         """Return a dictionary of stats information in Solr format or None
         on error."""
-        if self._result_cache is not None:
-            qr = QueryResponse(self._result_cache)
-            return qr.stats
+        if self._result_cache:
+            return self._result_cache.stats
+
         query_opts = self.query_opts()
         query_opts['rows'] = 0
         query_opts['hl'] = False
 
-        response = self.solr.query(**query_opts)
+        response = self.solr.query(rows=0, hl=False)  # **query_opts)
```

Comment:
what's this comment at the end of the line for?

Reply:
wow, thanks for catching that — I was testing to confirm that I could just pass in the overrides instead of generating query opts and overriding explicitly. Looks like I never cleaned up the old code!